avm/res/recovery-services/vault/main.bicep

```
@@ -240,22 +235,14 @@ module rsv_backupConfig 'backup-config/main.bicep' = if (!empty(backupConfig)) {
   name: '${uniqueString(deployment().name, location)}-RSV-BackupConfig'
   params: {
     recoveryVaultName: rsv.name
-    name: contains(backupConfig, 'name') ? backupConfig.name : 'vaultconfig'
-    enhancedSecurityState: contains(backupConfig, 'enhancedSecurityState')
-      ? backupConfig.enhancedSecurityState
-      : 'Enabled'
-    resourceGuardOperationRequests: contains(backupConfig, 'resourceGuardOperationRequests')
-      ? backupConfig.resourceGuardOperationRequests
-      : []
-    softDeleteFeatureState: contains(backupConfig, 'softDeleteFeatureState')
-      ? backupConfig.softDeleteFeatureState
-      : 'Enabled'
-    storageModelType: contains(backupConfig, 'storageModelType') ? backupConfig.storageModelType : 'GeoRedundant'
-    storageType: contains(backupConfig, 'storageType') ? backupConfig.storageType : 'GeoRedundant'
-    storageTypeState: contains(backupConfig, 'storageTypeState') ? backupConfig.storageTypeState : 'Locked'
-    isSoftDeleteFeatureStateEditable: contains(backupConfig, 'isSoftDeleteFeatureStateEditable')
-      ? backupConfig.isSoftDeleteFeatureStateEditable
-      : true
+    name: backupConfig.?name ?? 'vaultconfig'
+    enhancedSecurityState: backupConfig.?enhancedSecurityState ?? 'Enabled'
+    resourceGuardOperationRequests: backupConfig.?resourceGuardOperationRequests ?? []
+    softDeleteFeatureState: backupConfig.?softDeleteFeatureState ?? 'Enabled'
+    storageModelType: backupConfig.?storageModelType ?? 'GeoRedundant'
+    storageType: backupConfig.?storageType ?? 'GeoRedundant'
+    storageTypeState: backupConfig.?storageTypeState ?? 'Locked'
+    isSoftDeleteFeatureStateEditable: backupConfig.?isSoftDeleteFeatureStateEditable ?? true
```

```suggestion
    name: backupConfig.?name
    enhancedSecurityState: backupConfig.?enhancedSecurityState
    resourceGuardOperationRequests: backupConfig.?resourceGuardOperationRequests
    softDeleteFeatureState: backupConfig.?softDeleteFeatureState
    storageModelType: backupConfig.?storageModelType
    storageType: backupConfig.?storageType
    storageTypeState: backupConfig.?storageTypeState
    isSoftDeleteFeatureStateEditable: backupConfig.?isSoftDeleteFeatureStateEditable
```
Redundant as anyways the default. Ref: https://github.com/alexanderojala/bicep-registry-modules/pull/2